spec/features/users/regenerate_personal_key_spec.rb

```diff
@@ -4,6 +4,8 @@
   include XPathHelper
   include PersonalKeyHelper
   include SamlAuthHelper
+  include JavascriptDriverHelper
```

Comment:
is this needed? Looks like it provides the `javascript_enabled?` method which I'm not seeing used here

Reply:
Oops, this was me trying to resize things in an after action which didn't work